public/utils/validate.js

```diff
@@ -30,6 +30,11 @@ export const validateActionName = trigger => value => {
   if (matches.length > 1) return 'Action name is already used';
 };
 
+export const validateActionThrottle = action => value => {
```

Comment:
Few issues with this:

1. This is hard to read: (!action.throttle || !action.throttle.value)
Can we change this to:
_.get(action, 'throttle.value')

Same with the  if (action.throttle && action.throttle.value) below

2. If you fill out the throttle and then uncheck the throttle_enabled, it seems like this will still validate the throttle which is incorrect since it's not enabled.

3. If throttling is enabled and they put in 0, it'll say "Need to set throttle value" which is wrong.

Reply:
Will change to use _get(action, 'throttle.value')
Even the throttle_enabled = false, we should make sure the throttle value is not negative or 0, backend code requires the throttle value >0

Reply:
It doesn't make sense to validate on fields that are not enabled anymore. If throttle_enabled is false then the input should be disabled and when creating/updating the action the throttle should be set to null.

Reply:
Thanks Drew. Good suggestion. Fix it.